pom.xml

```diff
@@ -33,7 +33,7 @@
 
     <groupId>org.jenkins-ci.plugins.pipeline-stage-view</groupId>
     <artifactId>parent-pom</artifactId>
-    <version>1.2-SNAPSHOT</version>
+    <version>1.2-beta-2-SNAPSHOT</version>
```

Comment:
Why?

Changing versions around is fine, if you prefer to use `--batch` when running releases rather than typing in custom numbers, but best not to do as part of a feature PR.